### DISCUSSION TITLE
Documentation: Gradle works only with Java 1.8

### DISCUSSION THREAD

**Discussion by arky** [digidem/mapeo-mobile#831] [Ideas]
The mapeo-mobile android build toolchain: gradle only seem to work with OpenJDK 8. Using any other versions results in errors as mentioned here 
https://github.com/gradle/gradle/issues/8436

**Comment by gmaclennan** [comment]
Hi @arky, sorry about this, the docs should refer people to the [React Native setup guide](https://facebook.github.io/react-native/docs/getting-started) following the instructions for "React Native CLI Quickstart". I'll update them when I get a chance.
